automation/build-bin.ts

```diff
@@ -24,7 +24,7 @@ execPkg(['--target', 'host', '--output', 'build-bin/resin', 'package.json'])
 	.then(nativeExtensions => {
 		console.log(`\nCopying to build-bin:\n${nativeExtensions.join('\n')}`);
 
-		return nativeExtensions.map(extPath => {
+		return nativeExtensions.map((extPath: string) => {
```

Comment:
It strikes me as odd that the type of `extPath` can't be inferred here. Are you sure the type annotation is necessary? Isn't the type of `nativeExtensions` `string[]`?